code/build/template/build/premake4.lua

```diff
@@ -107,6 +111,12 @@ solution "[name]"
         "NoMinimalRebuild",
         "Symbols"
     }
+[+python
+
+    defines
+    {
+        "PK_ENABLE_OS=0"

```

Comment:
This leaves out some system interaction modules and functions from pocketpy. This could be changed by a user locally, but leaving IO to orx seems like a good default.